leshan-server-demo/webapp2/src/App.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div id="app">
+    <v-app>
+      <leshan-nav-bar/>
+      
+      <v-main>
+        <v-container fluid class="pa-0">
+          <router-view >  </router-view>  
+        </v-container>
+      </v-main>
+      <!--leshan-footer /-->
+    </v-app>
+  </div>
+</template>
+
+<script>
+import LeshanNavBar from "@/components/LeshanNavBar.vue";
+//import LeshanFooter from "@/components/LeshanFooter.vue";
```

Comment:
Commented code

Reply:
I hesitated to keep a footer but finally I think it take to much space and the about pages is enough.
I will remove all comments/files about footer.